src/com/rcv/ElectionConfig.java

```diff
@@ -145,6 +145,16 @@ public String visualizerOutput() {
     return rawConfig.visualizerOutput;
   }
 
+  // function: continueTabulationTillAllCandidatesEliminated
+  // purpose: getter for setting to keep tabulating beyond selecting winner(s) till all candidates
+  // have been eliminated
+  // returns: weather to keep tabulating past selecting all winner(s)
+  public Boolean continueTabulationTillAllCandidatesEliminated() {
```

Comment:
This can be a primitive. (I think it's helpful to be explicit in cases like this because it clarifies the contract for the API in terms of whether the return value is nullable.)